book/website/communication/peer-review/peer-review-conflicts.md

```diff
@@ -0,0 +1,84 @@
+The Forum agreed that the reviewer did have an interest which should have been declared. They should either have raised this with the editor, allowing the journal to make a decision on how to handle the situation, or declined to write a review.
+The editors should dismiss this review and seek a replacement if their policy is to have three reviews per paper, or go ahead with the two remaining reports. They should also provide some education for the reviewer on conflicts of interest.
+
+##### Follow-up
+The editors decided that the reviewer had made an honest mistake and have closed the case.
+
+### Examples in Open Peer Review: 
+There are also examples of COI in open peer review, where we could find other types of conflicts that occur and make them accountable.
+We provide below some examples in the open-source community due to co-authorship and shared affiliations:
+
+* Journal of Open Source Education: a reviewer of an [education-focused resource](https://github.com/openjournals/jose-reviews/issues/259#issuecomment-2393882259) indicated a co-authorship on a large author line paper and a little more interaction with the author through a large multi-investigator/multi-institutional effort. The editor-in-chief waived the conflict stressing the scope and open peer-review nature of the journal towards a more constructive and less gatekeeping goal. 
+* Environmental Data Science Book (EDS book): a reviewer of a notebook submission raised a [potential COI](https://github.com/alan-turing-institute/environmental-ds-book/issues/236#issuecomment-2021527918) because of a shared affiliation between the editor-in-chief who acted as an editor and authors of an open-source tool showcased in the notebook. The immediate follow-up was assigning a different editor to handle the revision of the submission.
+
+### Examples of COI policies
+Journals usually indicate their COI policies through a dedicated section on ethical guidelines/declarations. 
+COI policies largely vary by domain and community. 
+Some journals have specific requirements about what interests must be disclosed and the date range that you need to include (for example conflicts over the past 3 years). 
+The table below provides an example of some policies in peer-reviewed journals and other sectors for different research outputs.
+
+| Journal / Initiative                                             | Domain  | Research product     | Policy                                                                                                                                                                                                                                                                                                                         | Specific requirements (time)                                                         |
```

Comment:
I would here remove the policy column and add the links to the policy in the first column to the names of the initiatives: right now the links are quite distracting because it is a lot of text, and you already know that the policies are present as you write above: the table below provides an example of some policies. 
Let me know if that doesn't make any sense!